libzhl/functions/Entity.zhl

```diff
@@ -11,29 +11,12 @@ static void Entity::PlaySound(int ID, float Volume, int FrameDelay, bool Loop, f
 
 "558bec83e4f8515356578bd9e8????????84c0":
 __thiscall void Entity::CopyStatusEffects();
-"558bec83e4f8515356578b7d??8bf157e8????????84c00f85????????57":
-__thiscall void Entity::AddBleeding(const EntityRef& ref, int duration);
-
-"558bec83e4f883ec10568b75??5756":
-__thiscall void Entity::AddShrink(const EntityRef& ref, int duration);
-
-"558bec83e4f883ec38a1????????33c4894424??56578b7d??8bf1":
-__thiscall void Entity::AddMagnetized(const EntityRef& ref, int duration);
-
-"558bec83e4f883ec30a1????????33c4894424??56578b7d??8bf157e8????????84c00f85????????57ff75??8bcee8????????8bce8bd0":
-__thiscall void Entity::AddBaited(const EntityRef& ref, int duration);
-
-"558bec83ec30a1????????33c58945??56578b7d??8bf157e8????????84c00f85????????57ff75":
-__thiscall void Entity::AddBrimstoneMark(const EntityRef& ref, int duration);
-
-"558bec83ec30a1????????33c58945??56578b7d??8bf157e8????????84c00f85????????576afd":
-__thiscall void Entity::AddWeakness(const EntityRef& ref, int duration);
 
 "558bec5156ff75":
 __thiscall void Entity::AddIce(const EntityRef& ref, int duration);
 
 "558bec83e4f85156ff75??8bf1e8":
-__thiscall void Entity::AddKnockback(const EntityRef& ref, const Vector& pushDirection, int duration, bool takeImpactDamage);
+__thiscall void Entity::AddKnockback(EntityRef& ref, const Vector& pushDirection, int duration, bool takeImpactDamage);
```

Comment:
Any reason why AddKnockback does not take a const reference to the EntityRef ?

Reply:
Probably cheap solution to having trouble with my AddKnockback hook failing signature checks on game launch. I should be able to revert this.